compose-builder/add-mqtt-messagebus.yml

```diff
@@ -38,12 +38,10 @@ services:
 
   command:
     environment:
-      MESSAGEBUS_INTERNAL_TYPE: mqtt
-      MESSAGEBUS_INTERNAL_PROTOCOL: tcp
-      MESSAGEBUS_INTERNAL_HOST: edgex-mqtt-broker
-      MESSAGEBUS_INTERNAL_PORT: 1883
-      MESSAGEBUS_INTERNAL_AUTHMODE: none
-      MESSAGEBUS_EXTERNAL_CLIENTID: core-command-ex
+      MESSAGEBUS_OPTIONAL_CLIENTID: core-command
+      MESSAGEBUS_AUTHMODE: none
+      MESSAGEBUS_HOST: edgex-mqtt-broker
+      EXTERNALMQTT_CLIENTID: core-command-ex
```

Comment:
Port and Type should be kept.

Reply:
Yep, missed that the `mqtt-bus.env` file was not included on this service. Added it.